src/components/elements/group-item.astro

```diff
@@ -0,0 +1,44 @@
+---
+import type { CollectionEntry } from 'astro:content'
+import { Image, Picture } from "astro:assets";
+import Tag from "../../components/elements/tag.astro";
+import PostCategoryTag from "../../components/post-category-tag.astro";
+import Avatar from "../../components/avatar.astro";
+import Link from "./link.astro";
+
+interface Props {
+  group: CollectionEntry<'group'>;
+  index: number;
+}
+
+const { group, index } = Astro.props;
+
+---
+
+<li>
+  <a href={`/community/${group.slug}`}>
+    <div class="grid md:grid-cols-2 gap-2 md:gap-2 items-center py-2">
+      <Image
+        src={group.data.icon}
+        alt="Group icon for " + {group.data.name}
+        sizes="(max-width: 150) 150px, 150px"
+        style="object-fit: cover; border-radius: 50%; float:left"
```

Comment:
Could try:
class="rounded-full object-cover float-left"

In general, avoid float - should be possible to position using the grid, or use css Flexbox